passportConfig.ts

```diff
@@ -1,10 +1,11 @@
-const JwtStrategy = require("passport-jwt").Strategy;
-const ExtractJwt = require("passport-jwt").ExtractJwt;
-const AnonymousStrategy = require("passport-anonymous");
-const models = require("./models");
-const config = require("../../config");
+import { ExtractJwt, Strategy as JwtStrategy } from "passport-jwt";
+import AnonymousStrategy from "passport-anonymous";
+import models from "./models";
 
-module.exports = function(passport) {
+// @ts-ignore - Assume that this references some out of repo config?
```

Comment:
isn't this referring to config.js in the root of the repo?

Reply:
Would that not be just `import config from "./config` rather than `import config from "../../config"`?

Reply:
Right you are. I've just looked more closely and it seems like passportConfig isn't used at all. The function names in it aren't referenced anywhere and the tests pass when the file is removed. 

Please delete it in your PR.